routes/payment.js

```diff
@@ -42,31 +56,47 @@ router.post('/create-session', async (req, res) => {
       month: parseDateOne.getUTCMonth() + 1,
       year: parseDateOne.getUTCFullYear(),
     };
+
     const dateTwo = {
       day: parseDateTwo.getUTCDate(),
       month: parseDateTwo.getUTCMonth() + 1,
       year: parseDateTwo.getUTCFullYear(),
     };
 
+    const stripeId = user.stripeId;
+    const guestFee = reservationDetails.guestFee;
+    const listingPrice = listingDetails.price;
+
+
+    // Total price of the listing including guest fee. Host fee + our profit is taken from this. 
+    const listingTotal = (listingPrice * days * 100) + (guestFee * 100);
+
+    // Price we make - Host Fee (1%) + Guest Fee (10%)
+    const applicationFee = (listingPrice * days * 100) * 0.01 + (guestFee * 100)
```

Comment:
Line 72 and 75 - we should make these constants into global variables so we don't have any magic numbers.

Correct me if I'm wrong:
 - 0.01 represents HOST_FEE
 - 100 represents CENTS_TO_DOLLARS